### DISCUSSION TITLE
Stability Days not working for Docker images in Helm Values

### DISCUSSION THREAD

**Discussion by cdloh** [renovatebot/renovate#17069] [Request Help] [answered]
I have a [renovate config](https://gist.github.com/cdloh/c14bcdacad99c329a903b09d41be56f8) that I'm running over a private Flux repository

The layout of the repository is the same as an example repo [here](https://github.com/dfroberg/cluster/tree/main/cluster).

The problem I am having is that when I enable autoMerge to true stability days are not respected or set for merges for Helm values for docker images. Example MR here (MR is exactly the same in the repo I'm running this against ) - https://github.com/dfroberg/cluster/pull/1866

Am I doing anything wrong here?

**Comment by viceice** [comment, marked as answer]
docker datasource doesn't have release timestamp, so stability days won't work for docker images.

we won't change that, search the discussions, we've already answered that a couple of times.

**Reply by cdloh** [reply]
Sorry @viceice I searched "docker stability" and didn't find anything along those lines.

Why isn't the `org.opencontainers.image.created` label used for stability days?

**Reply by viceice** [reply]
was an isssue, maybe we discussed it on slack

- #5773

**Reply by viceice** [reply]
> Sorry @viceice I searched "docker stability" and didn't find anything along those lines.
> 
> Why isn't the `org.opencontainers.image.created` label used for stability days?

it would be an `o(n)`effort to fetch every config object for each docker tag. to expensive for renovate.

**Reply by cdloh** [reply]
Thanks @viceice appreciate the responses! Makes sense.
